hoomd/md/many_body.py

```diff
@@ -0,0 +1,402 @@
+import hoomd
+from hoomd.md import _md
+from hoomd.md.pair import Pair
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+
+
+class Triplet(Pair):
+    """Common three body potential documentation.
```

Comment:
This docstring appears to be copied directly from `Pair`. Much (maybe all?) of this is inaccurate. Triplet potentials have a different functional form compared to pair potentials. Also, add the attribute documentation for the `r_cut` type parameter.